lnbits/extensions/admin/models.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Optional
+
+from fastapi import Query
+from pydantic import BaseModel
+
+
+class UpdateSettings(BaseModel):
```

Comment:
- too many properties in one place, they should be grouped/nested

```
class UI(BaseModel):
    theme: str = Query(None)
    custom_logo: str = Query(None)
    ad_space: str = Query(None)
   ...


class LNbitsWallet(BaseModel):
    lnbits_endpoint: str = Query(None)
    lnbits_key: str = Query(None)

class EclairWallet(BaseModel):
    eclair_url: str = Query(None)
    eclair_pass: str = Query(None)

class FundingSources(BaseModel):
     lnbits: Optional[LNbitsWallet]
     eclair: Optional[EclairWallet]
     lnf_rest: Optional[EclairWallet]
     ...

...
class UpdateSettings(BaseModel):
    ui: UI
    funding_source: FundingSources
```

related to: https://github.com/lnbits/lnbits/pull/553#discussion_r1038323720